storage_drivers/eseries_iscsi.go

```diff
@@ -90,16 +92,16 @@ func (d *ESeriesStorageDriver) Validate() error {
 	log.Debugf("ESeriesStorageDriver#Validate()")
 
 	//Make sure the essential information was specified in the json config
-	if d.config.WebProxy_Hostname == "" {
-		return fmt.Errorf("WebProxy_Hostname is empty! You must specify the host/IP for the Web Services Proxy.")
+	if d.config.WebProxyHostname == "" {
+		return fmt.Errorf("WebProxyHostname is empty! You must specify the host/IP for the Web Services Proxy.")
 	}
 
-	if d.config.Controller_A == "" || d.config.Controller_B == "" {
-		return fmt.Errorf("Controller_A or Controller_B are empty! You must specify the host/IP for the E-Series storage array. If it is a simplex array just specify the same host/IP twice.")
+	if d.config.ControllerA == "" || d.config.ControllerB == "" {
+		return fmt.Errorf("ControllerA or ControllerB are empty! You must specify the host/IP for the E-Series storage array. If it is a simplex array just specify the same host/IP twice.")
 	}
 
 	if d.config.HostData_IP == "" {
```

Comment:
Possibly change hostData_IP to hostDataIP to make it consistent with some of the other variable name changes you have done.